test/libsolidity/syntaxTests/types/magic_block_basefee_error.sol

```diff
@@ -2,8 +2,14 @@ contract C {
     function f() public view returns (uint) {
         return block.basefee;
     }
+    function g() public view returns (uint ret) {
+        assembly {
+            ret := basefee()
+        }
+    }
 }
 // ====
-// EVMVersion: <=berlin
+// EVMVersion: =berlin
```

Comment:
We don't have to revert to `=` here. `<=` is better,
```suggestion
// EVMVersion: <=berlin
```

Reply:
Oh, actually let's leave it as is. I guess this part of the error message will be different for different VMs making the test fail: `(you are currently compiling for "berlin").`